sdk/python/feast/feature_view.py

```diff
@@ -50,7 +50,8 @@ class FeatureView:
     ttl: Optional[timedelta]
     online: bool
     input: DataSource
-
+    batch_source: Optional[DataSource] = None
```

Comment:
One more question. Is there a reason `batch_source` is Optional?

Reply:
Backwards compatibilty - users are currently using `input`. We should make `batch_source` mandatory  when we're going to make a version bump. wdyt?